packetbeat/_meta/kibana/6/dashboard/Packetbeat-tls.json

```diff
@@ -17,20 +17,20 @@
                         }
                     }
                 }, 
-                "title": "Navigation", 
+                "title": "Navigation-7x 7.x", 
```

Comment:
There seems to be some artifact of prior runs

Reply:
When running locally, it causes the same problem, so this seems to be a problem with the script as it is right now.

Reply:
Did a rerun. Made quite a bit of improvements during development which should have solved this one 👍